firmware/screen/src/ui/screen_launcher.cpp

```
@@ -0,0 +1,311 @@
+    lv_label_set_text(ctx->status_label, "DISCONNECTED");
+    lv_obj_align(ctx->status_label, LV_ALIGN_LEFT_MID, STATUS_DOT_DIA + 8, 0);
+
+    ctx->spinner = lv_spinner_create(pill, /*time=*/1000, /*arc length=*/60);
+    lv_obj_set_size(ctx->spinner, 22, 22);
+    lv_obj_align(ctx->spinner, LV_ALIGN_RIGHT_MID, 0, 0);
+    lv_obj_add_flag(ctx->spinner, LV_OBJ_FLAG_HIDDEN);
+}
+
+}  // namespace
+
+extern "C" lv_obj_t* launcher_create(void) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)